CONTRIBUTING.md

```diff
@@ -39,19 +39,37 @@ This subfolder contains some example `.rst` files that show how to implement spe
 ------
 ## Releases
 
-1. Bump the version:
-   1. `git pull main`
-   2. Create a new branch
+We use semantic versioning. When starting a new minor release series like `1.11`, we create a new Git branch `2.11` and then selectively cherry-pick any bug fixes from `main` into that release.
```

Comment:
is this meant to be "we create a new git branch `1.11`? not `2.11`?

Also, could we add this link for semantic versioning definition: https://semver.org/

Reply:
Oops! Pantsbuild is on v2, so I have the muscle memory of typing `2.x` all the time 😅